refactor/examples/basic.cairo

```diff
@@ -0,0 +1,10 @@
+fn main() -> felt252 {
+    42
+}
+
+fn is_zero(x: felt252) -> felt252 {
+    match x {
+        0 => 0,
+        _ => 1,
+    }
+}
```

Comment:
This seems inverted. Wouldn't an `is_zero` function return `1` when it __is__ `0`?

Reply:
Fixed.